pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -294,7 +294,7 @@ public BrokerResponse handleRequest(JsonNode request, @Nullable RequesterIdentit
     if ((offlineTableName == null) && (realtimeTableName == null)) {
       // No table matches the request
       LOGGER.info("No table matches for request {}: {}", requestId, query);
-      requestStatistics.setErrorCode(QueryException.BROKER_RESOURCE_MISSING_ERROR_CODE);
+      requestStatistics.setErrorCode(QueryException.TABLE_DOES_NOT_EXIST_ERROR_CODE);
```

Comment:
@Jackie-Jiang Given the changes in this PR should `BrokerMeter.RESOURCE_MISSING_EXCEPTIONS` still be used here?